src/css/copy-address-field.scss

```diff
@@ -0,0 +1,93 @@
+.copy-address-field {
+    .copy-address {
```

Comment:
Why the double-nest here? Can we not just target `.copy-address` (or perhaps `address-copier` or similar) directly and remove the need for a `copy-address-field` class?

Reply:
I just wanted to wrap the hidden text area element in the component as well - I guess I could try moving that inside  `.copy-address`